Vision/AnalyzeFormV2/Readme.md

```diff
@@ -0,0 +1,413 @@
+---
+page_type: sample
+languages:
+- python
+products:
+- azure
+- azure-search
+- azure-cognitive-services
+name: Analyze form sample skill for cognitive search
+urlFragment: azure-analyzeform-sample
+description: This custom skill extracts specific fields from the results of a trained form recognition.
+azureDeploy: https://raw.githubusercontent.com/Azure-Samples/azure-search-power-skills/master/Vision/AnalyzeForm/azuredeploy.json
+---
+# Deployment
+
+[![Deploy to Azure](https://azuredeploy.net/deploybutton.svg)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure-Samples%2Fazure-search-power-skills%2Fmaster%2FVision%2FAnalyzeForm%2Fazuredeploy.json)
```

Comment:
This doesn't look right: it uses the "AnalyzeForm" dir instead of "AnalyzeFormV2", and you seem to be missing the file itself.